views/templates/steps/post-restore.html.twig

```diff
@@ -67,7 +67,7 @@
 
     {% if download_logs is defined %}
       <div class="content__section">
-        <a class="link" href="{{ download_logs.download_path }}" download="{{ download_logs.filename }}">
+        <a class="link" href="{{ download_logs.download_path }}" download="{{ download_logs.filename }}" data-au-tracking="Restore logs downloaded">
```

Comment:
These events could reuse the constants defined in a PHP class, to avoid duplicating the same event name in different places (Currently duplicated in LogsService.php)